src/Symfony/Component/AssetMapper/Command/DebugAssetMapperCommand.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\Command;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\AssetMapper\AssetMapperRepository;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Outputs all the assets in the asset mapper.
+ *
+ * @experimental
+ *
+ * @author Ryan Weaver <ryan@symfonycasts.com>
+ */
+#[AsCommand(name: 'debug:assetmap', description: 'Outputs all mapped assets.')]
+final class DebugAssetMapperCommand extends Command
+{
+    private bool $didShortenPaths = false;
+
+    public function __construct(
+        private readonly AssetMapperInterface $assetMapper,
+        private readonly AssetMapperRepository $assetMapperRepository,
+        private readonly string $projectDir,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addOption('full', null, null, 'Whether to show the full paths')
+            ->setHelp(<<<'EOT'
+The <info>%command.name%</info> command outputs all of the assets in
+asset mapper for debugging purposes!
+EOT
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $allAssets = $this->assetMapper->allAssets();
+
+        $pathRows = [];
+        foreach ($this->assetMapperRepository->allDirectories() as $path => $namespace) {
+            $path = $this->relativizePath($path);
```

Comment:
We could make `AssetMapperRepository::getRepositories()` able to return the relative paths with some bool argument or even a new method. No big deal though, fine as is

Reply:
That's true - we already have the projectRootDir there. I don't feel too strongly either way :)

Reply:
if only the debug command needs that, I would keep it in the bug command for now